torch/autograd/_functions/reduce.py

```diff
@@ -184,40 +184,41 @@ def __init__(self, k, dim=None):
 
 class Norm(Function):
 
-    def __init__(self, norm_type=2, dim=None):
-        super(Norm, self).__init__()
-        self.norm_type = norm_type
-        self.dim = dim
+    @staticmethod
+    def forward(ctx, input, norm_type=2, dim=None):
```

Comment:
Can you please change `norm_type` to `p`, so that it matches the name used in `Variable`?